src/app/pages/ArticlePage/experimentTopStories/helpers.tsx

```diff
@@ -233,20 +263,55 @@ const buildRequestUrls = ({
   };
 };
 
+const getQuerySelectors = ({ variantName }: { variantName?: VariantNames }) => {
+  if (!variantName) {
+    return {
+      view: `div[data-experiment-position='secondaryColumn'] > section[aria-labelledby='top-stories-heading']`,
+      click: `div[data-experiment-position='secondaryColumn'] a[aria-labelledby*='top-stories-promo']`,
+    };
+  }
+
+  return {
+    view: `div[data-experiment-position='articleBody${variantName}'] > section[aria-labelledby='top-stories-heading']`,
+    click: `div[data-experiment-position='articleBody${variantName}'] a[aria-labelledby*='top-stories-promo']`,
+  };
+};
```

Comment:
Non-blocking suggestion:
```suggestion
const getQuerySelectors = ({ variantName }: { variantName?: VariantNames }) => {
  const experimentPosition = variantName ? `articleBody${variantName}` : 'secondaryColumn';
  
  return {
    view: `div[data-experiment-position='${experimentPosition}'] > section[aria-labelledby='top-stories-heading']`,
    click: `div[data-experiment-position='${experimentPosition}'] a[aria-labelledby*='top-stories-promo']`,
  };
};
```

Reply:
Much nicer haha